src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Persone__name">
+        My name is
+        {' '}
+        {person.name}
+      </h2>
+      <p className="Person__age">
+        {person.age && (
+          <>
+            I am
+            {' '}
+            {person.age}
+          </>
+        )}
+      </p>
+      <p className="Person__partner">
+        {person.isMarried === false
+          ? 'I am not married'
+          : `${person.partnerName} is my
+          ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Create a separate variable for wife/husband
![image](https://user-images.githubusercontent.com/96409563/223213092-cd5306ef-5ef8-42b6-b23b-741dd826dd10.png)

Reply:
ok